Resources/Prototypes/Entities/Clothing/Masks/masks.yml

```diff
@@ -233,3 +233,17 @@
   - type: DiseaseProtection
     protection: 0.1
   - type: IdentityBlocker
+
+- type: entity
+  parent: ClothingMaskClown
+  id: ClothingMaskCluwne
+  name: cluwne face and hair
+  suffix: Not to be mapped
+  noSpawn: true
```

Comment:
noSpawn will remove it from the spawn menu, which is not ideal for items like this, simply remove that and replace the suffix with [Unremovable], mappers will know whether or not to use them (same for the rest of prots with that)

Reply:
all the cluwne items no longer have noSpawn but instead have the suffix: "Unremoveable"